tests/unit_test/validator/file_import_test.py

```
@@ -0,0 +1,106 @@
+
+        return CrudFileUploader(
+            bot="test_bot",
+            user="test_user",
+            type="crud_data",
+            overwrite=False,
+            collection_name="test_collection"
+        )
+
+    def test_validate(self, uploader_instance):
+        file_content = {
+            "file_content": ("Salesstore.csv", open("tests/testing_data/file_content_upload/Salesstore.csv", "rb"))
```

Ruff (0.12.2)

Use a context manager for opening files

(SIM115)